protocol/protobuf/profile_showcase.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+option go_package = "./;protobuf";
+package protobuf;
+
+message ProfileShowcaseCommunity {
+  string community_id = 1;
+  uint32 order = 2;
+}
+
+message ProfileShowcaseAccount {
+  string address = 1;
+  string name = 2;
+  string color_id = 3;
+  string emoji = 4;
+  uint32 order = 5;
+}
+
+message ProfileShowcaseCollectible {
```

Comment:
@dlipicar can you please check if there is a need to add any additional fields to assets and collectibles?

Reply:
not sure what the "uid" is. To identify any collectible you need three separate components:
- chainID
- contract address
- tokenID